app/dashboard/embed.py

```diff
@@ -101,6 +117,19 @@ def embed(request):
         return err_response
 
     try:
+        badge = request.GET.get('badge', False)
+        if badge:
+            open_bounties = Bounty.objects.current() \
+                .filter(
+                    github_url__startswith=repo_url,
+                    network='mainnet',
+                    idx_status__in=['open']
+                )
+
+            return HttpResponse(
+                SVG_BADGE.replace('N_BOUNTIES', str(len(open_bounties))),
```

Comment:
We'll want to update this to use `.count()` versus `len()` to avoid performance issues.